engine/org.eclipse.birt.report.engine.emitter.wpml/src/org/eclipse/birt/report/engine/emitter/wpml/IWordWriter.java

```diff
@@ -377,4 +377,19 @@ void writeContent(int type, String txt, IStyle style, IStyle inlineStyle, String
 	 * End page
 	 */
 	void endPage();
+
```

Comment:
Is this breaking the API?

Reply:
The challenge is that the internal handling of the word emitter is based on the IWordWriter for doc and docx.
But both implementations have another structure. Therefore I had to add the 2 methods on the original interface.
But my point of view is that we can have new methods on interface side with new versions.

Reply:
Added default methods